src/server/builds/node.js

```diff
@@ -69,7 +69,8 @@ function algoliasearch(applicationID, apiKey, opts) {
 }
 
 algoliasearch.version = require('../../version.js');
-algoliasearch.ua = 'Algolia for Node.js (' + algoliasearch.version + ')';
+algoliasearch.ua = 'Node.js (' + process.versions.node + '); ';
```

Comment:
```suggestion
```

Reply:
I don't understand? Are you proposing to delete it ?